x-pack/test/reporting_api_integration/services/scenarios.ts

```diff
@@ -142,10 +142,10 @@ export function createScenarios({ getService }: Pick<FtrProviderContext, 'getSer
       .set('kbn-xsrf', 'xxx')
       .send(job);
   };
-  const generatePdf = async (username: string, password: string, job: JobParamsPDFDeprecated) => {
+  const generatePdf = async (username: string, password: string, job: JobParamsPDFV2) => {
     const jobParams = rison.encode(job);
     return await supertestWithoutAuth
-      .post(`/api/reporting/generate/printablePdf`)
+      .post(`/api/reporting/generate/printablePdfV2`)
```

Comment:
This does need to change eventually, but I'm not sure we want to do it in this PR.